lib/actions/device.coffee

```diff
@@ -128,25 +128,40 @@ exports.register =
 		Examples:
 
 			$ resin device register MyApp
+			$ resin device register MyApp --uuid 83cb718aee3abe63d2e0a0016ea49399cdc5b3f9b879dbb01acf73f28f5085
+			$ resin device register MyApp --uuid 83cb718aee3abe63d2e0a0016ea49399cdc5b3f9b879dbb01acf73f28f5085 --device-api-key existingDeviceKey
```

Comment:
It's just using coffeelint, right? You should be able to add coffeelint ignore markers for individual lines. From http://www.coffeelint.org/:

```coffeescript
# coffeelint: disable=max_line_length
foo = "some/huge/line/string/with/embed/#{values}.that/surpasses/the/max/column/width"
# coffeelint: enable=max_line_length
```